### ISSUE TITLE
WebRTC and GetUserMedia support ?

### ISSUE THREAD

**Issue by Frulko** [crosswalk-project/crosswalk-ios#8]
Hi, I would know if you have added the WebRTC and GetUserMedia features on override of WKWebview or if these features are planned ? 

Thanks :)

**Comment by zliang7** [comment]
Yes, this is one of our todo item. But It's low priority.  I don't think the XWalkView is ready now to start to implement a complex extension such as WebRTC. WIth the limited resource currently we have, XWalkView is our focus in short term.

**Comment by jesusprubio** [comment]
Hi guys, we've implemented this Cordova plugin, maybe we could help :)
https://github.com/eface2face/cordova-plugin-iosrtc/

**Comment by knightfox1337** [comment]
Anyone got any updates on this problem?

**Comment by jondong** [comment]
@jesusprubio that is great you've enabled WebRTC through Cordova plugin, and @knightfox1337 you may have a try if it works with Cordova extension of Crosswalk iOS through CocoaPods: https://cocoapods.org/pods/crosswalk-extension-cordova to see if it works for you. Report us any issue you've met and I'll try to fix it.

**Comment by okpatil4u** [comment]
+1 for webrtc support with crosswalk

**Comment by urbaniak** [comment]
:+1:

Looking forward for it.

**Comment by sirgalleto** [comment]
Hi, do you know if in the actual version have this support?

**Comment by Titoine** [comment]
Still no news ?

**Comment by hk0i** [comment]
Is there any solution for WebRTC with Crosswalk? Has anyone tried using the cordova iosrtc plugin?

**Comment by sirgalleto** [comment]
If you need to work with a good WebRTC support use react-native...
